src/hooks/auth.ts

```diff
@@ -65,16 +67,30 @@ export const useAuth = ({ middleware }: { middleware?: any } = {}) => {
   const logout = async () => {
     await api.post('/api/logout');
     mutate(null);
+    destroyCookie(null, 'authToken');
```

Comment:
why is first param null to destroyCookie?

Reply:
It's similar to `setCookies`, the first parameter for the context. By passing `null` as a context parameter, it's indicate that the function is called from the client-side without any specific sever context.